tmt/cli.py

```diff
@@ -517,6 +517,9 @@ def tests_create(
 @click.option(
     '--nitrate / --no-nitrate', default=True,
     help='Import test metadata from Nitrate')
+@click.option(
+    '--polarion / --no-polarion', default=False,
+    help='Import test metadata from Polarion')
```

Comment:
```suggestion
    help='Import test metadata from Polarion.')
```

This applies for all options.